src/components/button/Button.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Button.module.css';
+
+
+const Button = ({children, type, disabled, ClickHandler}) => {
```

Comment:
Props dienen camelCase geschreven te worden, anders ziet React het als een component:
```suggestion
const Button = ({children, type, disabled, clickHandler}) => {
```